### PR TITLE
🎨 [Frontend] Study name to Tab

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

When a study is open, its name will be displayed on to the tab name. When it's closed the default name (${productDisplayName} + ${stackName}) will be shown again.

#### Bonus:
- [x] search workspaces

TabName

## Related issue/s

- related to https://github.com/ITISFoundation/osparc-issues/issues/1728
- closes https://github.com/ITISFoundation/osparc-issues/issues/1807

## Dev-ops checklist

- [ ] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by GitHK — APPROVED** [review]
👍

**Review by sanderegg — COMMENTED** [review]
ok

**Review by ignapas — APPROVED** [review]

**Review by jsaq007 — APPROVED** [review]
👍🏼

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/data/Resources.js [hunk 1]
```diff
@@ -368,7 +368,7 @@ qx.Class.define("osparc.data.Resources", {
           getPageSearch: {
             useCache: false,
             method: "GET",
-            url: statics.API + "/workspaces:search?offset={offset}&limit={limit}&text={text}&order_by={orderBy}"
+            url: statics.API + "/workspaces?offset={offset}&limit={limit}&filters={filters}&order_by={orderBy}"
```
Comment (matusdrobuliak66) [inline services/static-webserver/client/source/class/osparc/data/Resources.js:371]: :+1:

**Pull request merged**